source/math/FloatBox2D.ooc

```diff
@@ -187,4 +187,12 @@ FloatBox2D: cover {
 		}
 		This new(xMinimum, yMinimum, xMaximum - xMinimum, yMaximum - yMinimum)
 	}
+	linearInterpolation: static func (a, b: This, ratio: Float) -> This {
+		FloatBox2D new(
+			Float linearInterpolation(a left, b left, ratio),
+			Float linearInterpolation(a top, b top, ratio),
+			Float linearInterpolation(a width, b width, ratio),
+			Float linearInterpolation(a height, b height, ratio)
+		)
```

Comment:
There may be more efficient ways of computing it, though. Maybe 
```ooc
FloatBox2D createFrom(FloatPoint2D linearInterpolation(a leftTop,  b leftTop), FloatSize2D linearInterpolation(a size, b size))
```
works too, I'm a bit too tired to do the math right now. If you'd added a test for this function in this PR, it'd be easy to find out.